Resources/fsh-generated/resources/OperationDefinition-ISiKAppointmentBookOperation.json

```diff
@@ -24,15 +24,15 @@
       "use": "in",
       "min": 1,
       "max": "1",
-      "documentation": "Eine Appointment-Ressource entsprechend dem dazugehörigen ISiK-Profil MUSS übergeben werden. Falls das Parameter-Element 'Appointment' als einziges Parameter-Element verwendet wird, SOLL die Appointment-Ressource direkt im HTTP Body übergeben werden, alternativ zu einer Parameter-Ressource inklusive dem eingebetteten Appointment-Element. Invalide Ressourcen MÜSSEN abgelehnt werden durch das Termin Repository (zur Form der Ablehnung s.u.). Das Appointment MUSS den Status 'proposed' enthalten.",
+      "documentation": "Eine Appointment-Ressource entsprechend dem dazugehörigen ISiK-Profil MUSS übergeben werden. Falls das Parameter-Element 'Appointment' als einziges Parameter-Element verwendet wird, SOLL die Appointment-Ressource direkt im HTTP Body übergeben werden, alternativ zu einer Parameter-Ressource inklusive dem eingebetteten Appointment-Element. Invalide Ressourcen MÜSSEN abgelehnt werden durch das Termin Repository (zur Form der Ablehnung s.u.). Das Appointment MUSS den Status 'proposed' enthalten. Bei Terminverschiebungen (keine Terminabsage!) kann ein angepasstes Appointment in diesem Parameter übergeben werden. Appointment-Ressourcen die dem Termin-Repository bekannt sind unter der im Appointment enthaltenen id MÜSSEN durch das Termin-Repository geupdatet werden unter Beibehaltung der id.",
       "type": "Appointment"
     },
     {
       "name": "cancelled-appt-id",
       "use": "in",
       "min": 0,
       "max": "1",
-      "documentation": "Für die vorliegende Spezifikation ist die Verschiebung eines Termin eins zwei-stufiger Prozess, bei dem ein Termin storniert und ein neuer Termin neu gebucht wird. Dieser Parameter repräsentiert die Ressourcen-Id des stornierten Appointments. Der uri-Parameter kann eine absoulte URL enthalten, Server SOLLEN jedoch nur Termine für ihre eigene Domäne verwalten. Im neu-angelegten Appointment MUSS eine Reference auf den abgesagten Termin hinterlegt werden (vgl. Appointment.extension:replaces). Der Status der abgesagten Ressource MUSS durch den Server angepasst werden.",
+      "documentation": "Für die vorliegende Spezifikation ist die Absage eines Termin ein zwei-stufiger Prozess, bei dem ein Termin storniert und ein neuer Termin neu gebucht wird. Dieser Parameter repräsentiert die Ressourcen-Id des stornierten Appointments. Der uri-Parameter kann eine absoulte URL enthalten, Server SOLLEN jedoch nur Termine für ihre eigene Domäne verwalten. Im neu-angelegten Appointment MUSS eine Reference auf den abgesagten Termin hinterlegt werden (vgl. Appointment.extension:replaces). Der Status der abgesagten Ressource MUSS durch den Server angepasst werden.",
```

Comment:
Hier geht es um die Verschiebung und nicht um eine Absage oder?

Reply:
Schließe mich dem Kommentar an, folgender Satz ist in seiner gegenwärtigen Form verwirrend:
"_Für die vorliegende Spezifikation ist die Absage eines Termin ein zwei-stufiger Prozess, bei dem ein Termin storniert und ein neuer Termin neu gebucht wird._"
-> Wenn ein Termin abgesagt werden soll, wieso wird dabei dann ein neuer Termin gebucht?
Das mit der Storno&Neu-Anlage trifft ja nur bei einer der Terminverschiebungsvarianten zu, falls ein KIS diese Business-Logik unterstützt.

Reply:
@alexzautke ich denke, hier wurde nur "Verschiebung" mit "Absage" vertauscht bzw. Aktualisierung, wie man es bezeichnen will.

Reply:
@regelkraut @markus020 Danke, neue Formulierung ist hier: https://github.com/gematik/spec-ISiK-Terminplanung/pull/177

In https://github.com/gematik/spec-ISiK-Terminplanung/pull/170/files hatte Verschiebung schon zu Absage geändert, die Änderung war jedoch noch nicht in diesem Branch drin.